api/src/main/java/org/openmrs/module/ugandaemrreports/reports/SetupActiveOnCareList.java

```diff
@@ -165,6 +166,14 @@ public ReportDefinition constructReportDefinition() {
         addColumn(dsd,"latestViralLoad",hivPatientData.getViralLoadByEndDate());
         addColumn(dsd,"latestVLQualitative",hivPatientData.getVLQualitativeByEndDate());
         addColumn(dsd,"returnVisitDate",hivPatientData.getLastReturnDateByEndDate());
+        addColumn(dsd,"model",hivPatientData.getDSDMModel());
+        dsd.addColumn("DSDM Date",new DSDMModelDataDefinition(), "", df.getDateEnrolledConverter());
```

Comment:
@dbaluku Why does this one have dsd prefix

Reply:
it implemented same way  as birth date